docs/content/development/entities/create.md

```diff
@@ -136,8 +136,11 @@ You can access your custom entity data in the database in services or subscriber
 
 ```ts
 import { TransactionBaseService } from "@medusajs/medusa"
+import PostRepository from "../repositories/post";
 
 class PostService extends TransactionBaseService {
+  protected readonly postRepository: typeof PostRepository;
```

Comment:
suggestion: We generally suffix with `_` to give the flexibility to have a getter and avoid any name conflict.

Reply:
sure